code/modules/mob/dead/observer/observer.dm

```diff
@@ -578,6 +579,46 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		M.antagHUD = 1
 		to_chat(src, "<span class='notice'><B>AntagHUD Enabled</B></span>")
 
+
+
+/mob/dead/observer/verb/toggle_pathogenHUD()
+	set category = "Ghost"
+	set name = "Toggle PathogenHUD"
+	set desc = "Toggles Pathogen HUD allowing you to see airborne pathogenic clouds, and infected items and splatters"
+	if(!client)
+		return
+	if(pathogenHUD)
+		pathogenHUD = FALSE
+		to_chat(src, "<span class='notice'><B>Pathogen HUD disabled.</B></span>")
+		science_goggles_wearers.Remove(src)
+		if (client)
+			for (var/obj/item/I in infected_items)
+				client.images -= I.pathogen
+			for (var/mob/living/L in infected_contact_mobs)
+				client.images -= L.pathogen
+			for (var/obj/effect/effect/pathogen_cloud/C in pathogen_clouds)
+				client.images -= C.pathogen
+			for (var/obj/effect/decal/cleanable/C in infected_cleanables)
+				client.images -= C.pathogen
+	else
+		pathogenHUD = TRUE
+		to_chat(src, "<span class='notice'><B>Pathogen HUD enabled.</B></span>")
+		science_goggles_wearers.Add(src)
+		if (client)
+			for (var/obj/item/I in infected_items)
+				if (I.pathogen)
+					client.images |= I.pathogen
```

Comment:
Any reason this is `|=` and not `+=` ? I think all of these pathogens are distincts

Reply:
This is to prevent one getting added twice through some obscure manipulation that'd cause one to remain after it gets disabled. Better safe than sorry. It's not that much more CPU costly.